### PR TITLE
CA-18 RDS Postgres Initial Tests

### PR DESCRIPTION
**Pull request by jamoroso-caylent**
Initial tests:

- creates the RDS instance
- waits to call the AWS API describe RDS function and asserts values are the same as outputs from terraform.

Also, this PR contains changes in other non-test related files due to running terraform fmt and make terraform/docs

### CODE REVIEW

File: examples/minimal/outputs.tf [hunk 1]
```diff
@@ -3,6 +3,30 @@ output "ingress_ports" {
   description = "List of ingress ports"
 }
 
-output "rds" {
-  value = module.rds_postgres
```
Comment (franco-martin) [inline examples/minimal/outputs.tf:7]: I prefer outputting the whole module, but lets see if @souza-dan agrees

**Review by franco-martin — COMMENTED** [review]
Looking good, lets see what dan thinks of the outputs

**Review by souza-dan — COMMENTED** [review]
Overall looks good!

**Review by franco-caylent — APPROVED** [review]
LGTM

**Review by souza-dan — APPROVED** [review]
LGTM

**Pull request merged**